frontend-react/src/hooks/network/WatersHooks.ts

```diff
@@ -0,0 +1,58 @@
+import { useMutation } from "@tanstack/react-query";
```

Comment:
fwiw I would maybe have moved this functionality into the `useFileHandler` hook somehow? but this is good too

Reply:
Along with some rearrangement of stuff post Deliveries react-query work, I want to go back through and assess _where_ some things are called, so I can add this to the list alongside how we handle the Report Details page in the Delivery work.

Reply:
For reference: https://github.com/CDCgov/prime-reportstream/pull/6977#discussion_r993768855